test/UnitTest/persistance/test_persistance.py

```diff
@@ -0,0 +1,128 @@
+import os,sys
```

Comment:
[minor]please try to cover the whole code. Full code coverage is required for each feature.